packages/data-transforms/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './trackValueArrayToSegments'
```

Comment:
I'd go for kebab-case in the names of the files to follow the same structure that the rest of the packages